packages/ckeditor5-dev-translations/lib/utils/getpackagecontext.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * @license Copyright (c) 2003-2024, CKSource Holding sp. z o.o. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+import upath from 'upath';
+import fs from 'fs-extra';
+import { CONTEXT_FILE_PATH } from './constants.js';
+
+/**
+ * @param {object} options
+ * @param {string} options.packagePath Path to the package containing the context.
+ * @returns {Context}
+ */
+export default function getPackageContext( { packagePath } ) {
+	const contextFilePath = upath.join( packagePath, CONTEXT_FILE_PATH );
+	const contextContent = fs.readJsonSync( contextFilePath, { throws: false } ) || {};
+
+	return {
+		contextContent,
+		contextFilePath,
+		packagePath
+	};
+}
+
+/**
+ * @typedef {object} Context
```

Comment:
```suggestion
 * @typedef {object} TranslationsContext
```
WDYT?

Reply:
Done in https://github.com/ckeditor/ckeditor5-dev/pull/1025/commits/5eb675f3874e4aefc83807afcfe12c182521eccd.